src/shared/utils/getRandomNumber.util.ts

```diff
@@ -0,0 +1,17 @@
+export function getRandomNumber({ min, max }: { min: number; max: number }): number {
```

Comment:
This is leftover from my playing around with code... We should delete this since it is not being used currently...